app/views/SaPrintSummaryPage.scala.html

```diff
@@ -0,0 +1,286 @@
+
+@maybeUpfrontPaymentAmount = @{
+    upfrontPaymentAmount.map { someAmount =>
+        SummaryListRow(
+            key = Key(
+                content = HtmlContent(Html(Messages.UpfrontPaymentSummary.`Upfront payment`.show)),
+                classes = "govuk-!-width-one-half"
+            ),
+            value = Value(
+                content = Text(someAmount.gdsFormatInPounds)
+            ),
+            actions = None
+        )
+    }.toList
+}
+
+@paymentDayRow = @{
+    SummaryListRow(
+        key = Key(
+            content = HtmlContent(Html(Messages.PaymentSchedule.`Payments collected on`.show)),
+            classes = "govuk-!-width-one-half"
+        ),
+        value = Value(
+            content = Text(s"${dayOfMonth.toString}${DateMessages.getSuffix(dayOfMonth)} ${Messages.PaymentSchedule.`or next working day`.show}")
+        ),
+        actions = None
+    )
+}
+
+@monthRows = @{
+    paymentPlan.collections.regularCollections.sortBy(_.dueDate.value).zipWithIndex.map { case (p, index) =>
+        SummaryListRow(
+            classes = s"grouped-row ${
+                if(index > (paymentPlan.collections.regularCollections.size - 2)) {
+                    "last-grouped-row"
+                } else {
+                    ""
+                }
+            }",
+            key = Key(
+                content = Text(s"${DateMessages.monthName(p.dueDate.value.getMonthValue).show} ${p.dueDate.value.getYear.toString}"),
+                classes = s"govuk-!-width-one-half${if(index === 0) " govuk-!-padding-top-2" else ""}"
+            ),
+            value = Value(
+                content = Text(p.amountDue.value.gdsFormatInPounds)
+            ),
+            actions = None
+        )
+    }
+}
+
+@interestRow = @{
+    SummaryListRow(
+        key = Key(
+            content = HtmlContent(Html(Messages.PaymentSchedule.`Estimated total interest`.show)),
+            classes = "govuk-!-width-one-half"
+        ),
+        value = Value(
+            content = Text(paymentPlan.planInterest.value.gdsFormatInPounds)
+        ),
+        actions = None
+    )
+}
+
+@totalRow = @{
+    SummaryListRow(
+        key = Key(
+            content = Text(Messages.PaymentSchedule.`Total to pay`.show),
+            classes = "govuk-!-width-one-half"
+        ),
+        value = Value(
+            content = Text(paymentPlan.totalDebtIncInt.value.gdsFormatInPounds)
+        ),
+        actions = None
+    )
+}
+
+
+@monthlyPaymentRows = @{ paymentDayRow :: monthRows ::: List(interestRow) ::: List(totalRow) }
+
+@layout(pageTitle = Some(title), additionalScripts = Some(scripts)) {
+
+    <h1 class="govuk-heading-xl">
+    @title
+    </h1>
+
+    <p class="govuk-body js-visible">
+        <a href="#print-dialogue" class="govuk-link" id="printLink">
+            @{
+            Messages.Confirmation.`Print a copy of your payment plan`.show
```

Comment:
I think this is not consistent with the other two regimes. You may want to raise it and make it consistent or correct this to match the prototype. It is meant to say `Print or save a copy of your payment plan`